public/app/features/dashboard-scene/sharing/ExportButton/ResourceExport.tsx

```diff
@@ -12,6 +12,7 @@ export enum ExportMode {
   Classic = 'classic',
   V1Resource = 'v1-resource',
   V2Resource = 'v2-resource',
+  V2TransformedToV1 = 'v2-transformed-to-v1',
```

Comment:
why is it a new target?  can we target `v1-resource`?  

and then it would need to be transformed if the source is v2 (identical to selecting v2 when the source is v1)